include/envoy/http/filter.h

```diff
@@ -418,6 +418,20 @@ class StreamDecoderFilterCallbacks : public virtual StreamFilterCallbacks {
   // Note that HttpConnectionManager sanitization will *not* be performed on the
   // recreated stream, as it is assumed that sanitization has already been done.
   virtual bool recreateStream() PURE;
+
+  /**
+   * Adds socket options to be applied to the upstream request. Note that unique values for the
```

Comment:
Options are on the connection right? Maybe "Options that should be applied to any connections used for upstream requests"? 

There can be more than one upstream request per downstream request (retries, hedging, etc.), so the wording should probably reflect that

Reply:
Makes sense.